resources/public/api/conf/2.0/schemas/income-sa-additional-information-response.json

```diff
@@ -9,12 +9,12 @@
       "properties": {
         "self": {
           "type": "object",
-          "description": "The URI to this resource",
+          "description": "The Uniform Resource Identifier (URI) for this endpoint.",
           "properties": {
             "href": {
               "type": "string",
               "description": "The URL of the endpoint.",
-              "example": "/individuals/income/sa/additional-information?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430{&fromDate,toDate}"
+              "example": "/individuals/income/sa/additional-information?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430{&fromTaxYear,toTaxYear}"
```

Comment:
Did we say we should include the date itself in schemas? Other than root HATOES links? 

https://github.com/hmrc/individuals-employments-api/blob/master/resources/public/api/conf/2.0/schemas/employments-response.json#L17

Example from employments ^^

Reply:
👍